### PR TITLE
[dev-menu] [ENG-10036] update dev menu to make refresh more prominent

### PR DESCRIPTION
**Pull request by kadikraman**
# Why

Refresh is the most often used button in the dev menu, but it is not the one that stands out the most.
This change will clean up the dev menu to separate out the refresh button and make it stand out more.

# How

- move the "copy link" button next to the URL (since that's the link you're copying)
- make the whole link section peessable (not just the link icon)
- move "home" next to the other buttons
- move "reload" next to the other buttons, but separate it out to its own section

# Test Plan

| Before    | After |
| -------- | ------- |
| Simulator Screenshot - iPhone 14 Pro - 2023-09-13 at 17 10 38  | Simulator Screenshot - iPhone 14 Pro - 2023-09-14 at 18 35 15    |

# Checklist

- [ ] Documentation is up to date to reflect these changes (eg: https://docs.expo.dev and README.md).
- [ ] Conforms with the [Documentation Writing Style Guide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md)
- [ ] This diff will work correctly for `npx expo prebuild` & EAS Build (eg: updated a module plugin).

### CODE REVIEW

File: packages/expo-dev-menu/app/components/Main.tsx [hunk 1]
```diff
@@ -210,7 +192,23 @@ export function Main({ registeredCallbacks = [], isDevice }: MainProps) {
...
+                  <SettingsRowButton
+                    label="Go home"
+                    icon={<HomeFilledIcon />}
+                    onPress={actions.navigateToLauncher}
+                  />
+                </View>
+                <Divider />
+              </>
+            )}
             <View roundedTop="large" bg="default">
```
Comment (gabrieldonadel) [inline packages/expo-dev-menu/app/components/Main.tsx:212]: Nit
image

```suggestion
            <View  bg="default">
```

**Review by gabrieldonadel — COMMENTED** [review]
Code looks good to me, you just need to update the embedded bundle (`yarn bundle:prod:ios && yarn bundle:prod:android`) and fix that nit

And thanks for updating expo-dev-menu's README.md.

**Review by brentvatne — APPROVED** [review]
👏

**Comment by kbrandwijk** [comment]
Layout seems slightly broken in the screenshot on the right. Toggle performance monitor should not have top rounded corners.

**Comment by kbrandwijk** [comment]
> Layout seems slightly broken in the screenshot on the right. Toggle performance monitor should not have top rounded corners.

Nvm, @gabrieldonadel already spotted it.

**Review by lukmccall — APPROVED** [review]

File: packages/expo-dev-menu/app/components/Main.tsx (hunk 1 above)
Comment (kadikraman) [reply packages/expo-dev-menu/app/components/Main.tsx:212]: Oops! I can't believe I missed this. Updated ✔️

**Review by gabrieldonadel — APPROVED** [review]

**Comment by brentvatne** [comment]
one nit - i think we should group "Reload" and "Go home" together, and then the "Toggle performance monitor" etc should be in a separate group. currently Reload is on its own and "Go home" is grouped with the other tools, but I think "Go home" conceptually is closer to "Reload" than the others

**Review by douglowder — APPROVED** [review]

**Comment by kbrandwijk** [comment]
> one nit - i think we should group "Reload" and "Go home" together, and then the "Toggle performance monitor" etc should be in a separate group. currently Reload is on its own and "Go home" is grouped with the other tools, but I think "Go home" conceptually is closer to "Reload" than the others

While conceptually closer, the whole point of the original ticket (to me) seemed to make reload stand out more than 'Go home' (and copy link, but that's solved differently now). That being said, I like the reload button in its standalone form like this, it takes a little less precision to hit it because it's surrounded by some margin.

Regardless of my opinion, if we keep it grouped, it needs bottom rounded corners.

**Comment by kadikraman** [comment]
@kbrandwijk fixed the rounded corners. Also made sure we support the case where `isDevLauncherInstalled=false`

| isDevLauncherInstalled=true    | isDevLauncherInstalled=false |
| -------- | ------- |
| Simulator Screenshot - iPhone 14 Pro - 2023-09-14 at 18 35 15 | Simulator Screenshot - iPhone 14 Pro - 2023-09-14 at 18 34 46    |

**Pull request merged**